modules/webgl/src/classes/shader.js

```diff
@@ -76,7 +76,11 @@ export class Shader extends Resource {
   }
 
   // PRIVATE METHODS
-  _compile() {
+  _compile(source = this.source) {
+    if (!source.startsWith('#version ')) {
+      source = `#version 100\n${source}`;
```

Comment:
Are we hitting this path? I.e. are there cases where we don't go through assembleShaders?

Reply:
Yes, if you create a `Program` object directions, assembleShaders isn't called. It's only called directly via `Model`.